frontend/src/cljs/imas_seamap/views.cljs

```diff
@@ -866,53 +868,180 @@
       :onChange #(re-frame/dispatch [:map/update-active-zone-iucn %])
       :keyfns
       {:id   :name
-       :text :name}}]]])
+       :text :name}}]]
+   [:a.data-coverage-report-link {:on-click open-data-coverage-report} "View data coverage report"]])
 
 (defn habitat-statistics
   []
-  (let [selected-tab (reagent/atom "breakdown")]
+  (let [selected-tab (reagent/atom "breakdown")
+        collapsed?   (reagent/atom false)]
     (fn []
-      (let [habitat-statistics @(re-frame/subscribe [:map/habitat-statistics])
+      (let [loading?           @(re-frame/subscribe [:map/habitat-statistics-loading?])
```

Comment:
I'm not too fussed; this keeps the code simple, possibly at the cost of polluting the subscription namespace a bit.  An alternative might be to have a "union type" for the habitat stats; they're either :habitat-stats/loading _or_ the data (or I guess :habitat-stats/error, we probably ignore that for now but you get the gist).  As I said, not fussed here, just raising it as something to consider as this app gets larger.